<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>743</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Zombies</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Juego">
    <property name="title">
     <string>&amp;Juego</string>
    </property>
    <addaction name="actionNewGame"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menu_Juego"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
  </widget>
  <action name="actionSalir">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="iconText">
    <string>Salir</string>
   </property>
   <property name="statusTip">
    <string>Terminar la aplicaci√≥n</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionNewGame">
   <property name="text">
    <string>&amp;Nuevo Juego</string>
   </property>
   <property name="iconText">
    <string>Nuevo Juego</string>
   </property>
   <property name="statusTip">
    <string>Comenzar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
